hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/fees/calculation/system/txns/UncheckedSubmitResourceUsage.java

```diff
@@ -42,4 +42,12 @@ public FeeData usageGiven(final TransactionBody txn, final SigValueObj sigUsage,
             throws InvalidTxBodyException {
         return FeeData.getDefaultInstance();
     }
+    /**
+     * This method is used to calculate the fee for the {@code NetworkUncheckedSubmit}
+     * query in modularized code only.
+     * @return the fee for the query
+     */
+    public static FeeData usageGiven() {
+        return FeeData.getDefaultInstance();
+    }
```

Comment:
Do we need this ? If we don't implement `calculateFees` in `UncheckedSubmitHandler` it takes default fees

Reply:
A comment on TransactionHandler.calculateFees reads "NOTE: FUTURE: This method should not be default, but should be implemented by all transaction handlers. This is a temporary measure to avoid merge conflicts." So therefore I think that calculateFees should be implemented in UncheckedSubmitHandler

Reply:
I see . Thanks